src/Analyzers/CSharp/Tests/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -4813,12 +4819,12 @@ interface IGoo
 
             public class C : IGoo
             {
-                void IGoo.Goo1(object x)
+                void IGoo.Goo1([IUnknownConstant, Optional] object x)
```

Comment:
i'm not undertstanding why we want these attributes.  can you explain why?

Reply:
I changed it so that explicitly implementing an interface method also carries over the parameters' attributes. This is better imo, for cases like `NotNull` attributes on nullable parameters. The user is still always free to remove those to their will.

Reply:
If it's better for those attributes, we shoudl only carry over those attributes (with explanations why).  We should not carry over the attributes that aren't appropriate to carry over.

Reply:
Will prefer not carrying over any attributes for the time being, as also mentioned in offline discussion. A future iteration involving carrying over important attributes on explicit interface implementations is welcome.

Reply:
Applied changes